normalize.js

```diff
@@ -114,7 +114,7 @@ define(['require', 'module'], function(require, module) {
     fromBase = removeDoubleSlashes(fromBase);
     toBase = removeDoubleSlashes(toBase);
 
-    var urlRegEx = /@import\s*("([^"]*)"|'([^']*)')|url\s*\(\s*(\s*"([^"]*)"|'([^']*)'|[^\)]*\s*)\s*\)/ig;
+    var urlRegEx = /@import\s*((?:"([^"]*)")|(?:'([^']*)'))|url\s*\(\s*(\s*(?:"([^"]*)")|(?:'([^']*)')|[^\)]*\s*)\s*\)/ig;
```

Comment:
@fearphage no worries. If u have a better idea, comment below, and I will try to apply yur suggestion if it works correctly with tests. I'm working on this so during this week, I may create another PR.